lab_6/task_6_3.py

```diff
@@ -0,0 +1,10 @@
+def create_list():
+    list_str = []
+    with open('C:/Users/Andrii/Desktop/learning_python.txt', 'r', encoding='utf-8') as lstr:
+        for __ in lstr:
+            list_str.append(lstr.readlines())
```

Comment:
so, in such way you create a list with one element -- list with lines.
just try this:
```suggestion
   
    with open('C:/Users/Andrii/Desktop/learning_python.txt', 'r', encoding='utf-8') as lstr:
         return lstr.readlines() 
```